src/app/@ansyn/overlays/services/overlays.service.ts

```diff
@@ -68,6 +68,7 @@ export class OverlaysService {
 	search(params: OverlaysCriteria): Observable<OverlaysFetchData> {
 		let feature = params.region;
 		return this._overlaySourceProvider.fetch({
+			dataInputFilters: Boolean(params.dataInputFilters) ? params.dataInputFilters : null,
```

Comment:
You don't need this, it's ok if `params.dataInputFilters` is undefined.

Reply:
I Do need this....application crash otherwise.